### PR TITLE
Course list item styling (easy bugfix)

### PR DESCRIPTION
**Pull request by heatsink**
Fix icon memory location & icon color (mistake in previous commit)

Forgot to specify the color..
Fix for https://github.com/mobileappdevhm/dev-team-1-cie-app-in-flutter/issues/52

### CODE REVIEW

**Comment by heatsink** [comment]
screen shot 2018-05-25 at 6 51 49 pm
Fix shown above, we have color and different icons again!

**Comment by RedTo** [comment]
Please add RGB and hex code to new defined colors and select a good name :-)

File: lib/generic/genericIcon.dart [hunk 1]
```diff
@@ -9,20 +9,24 @@ class GenericIcon {
   static const String fontFamily = 'MaterialIcons';
   static Icon buildGenericAvailabilityIcon(int availability) {
     IconData iconData;
+    Color color;
```
Comment (RedTo) [inline lib/generic/genericIcon.dart]: Can be deleted

File: lib/generic/genericIcon.dart [hunk 2]
```diff
@@ -9,20 +9,24 @@ class GenericIcon {
...
     switch (availability) {
       case CourseDefinitions.AVAILABILITY_AVAILABLE:
         iconData = const IconData(memStorageAvailable, fontFamily: fontFamily);
+        color = const Color.fromRGBO(111, 207, 151, 1.0);
```
Comment (RedTo) [inline lib/generic/genericIcon.dart]: color = CiEColor.green;

File: lib/generic/genericIcon.dart [hunk 3]
```diff
@@ -9,20 +9,24 @@ class GenericIcon {
...
         break;
       case CourseDefinitions.AVAILABILITY_PENDING:
-        iconData = const IconData(memStorageAvailable, fontFamily: fontFamily);
-        break;
+        iconData = const IconData(memStoragePending, fontFamily: fontFamily);
+        color = const Color.fromRGBO(242, 201, 76, 1.0);
```
Comment (RedTo) [inline lib/generic/genericIcon.dart]: color = CiEColor.yellow;

File: lib/generic/genericIcon.dart [hunk 4]
```diff
@@ -9,20 +9,24 @@ class GenericIcon {
...
+    break;
       case CourseDefinitions.AVAILABILITY_UNAVAILABLE:
-        iconData = const IconData(memStorageAvailable, fontFamily: fontFamily);
+        iconData = const IconData(memStorageUnavailable, fontFamily: fontFamily);
+        color = const Color.fromRGBO(235, 87, 87, 1.0);
```
Comment (RedTo) [inline lib/generic/genericIcon.dart]: color = CiEColor.red;

File: lib/generic/genericIcon.dart (hunk 1 above)
Comment (RedTo) [reply lib/generic/genericIcon.dart]: Forget my comment. ;-)

**Comment by heatsink** [comment]
👍 Thanks for the review guys

**Comment by RedTo** [comment]
Could this branch be deleted?

**Pull request merged**